src/styles/typography.ts

```diff
@@ -3,25 +3,25 @@ import { TextStyle } from "react-native"
 import * as Colors from "./colors"
 
 // Font Size
-export const xxxSmall = 11
-export const xxSmall = 12
-export const xSmall = 13
-export const small = 15
-export const medium = 17
-export const large = 19
-export const xLarge = 22
-export const xxLarge = 28
-export const huge = 52
+export const xxxSmall = 10
+export const xxSmall = 11
+export const xSmall = 12
+export const small = 13
+export const medium = 15
+export const large = 17
+export const xLarge = 19
+export const xxLarge = 24
+export const xxxLarge = 28
```

Comment:
Yesss love this naming change. Though I was hoping we might get to `Typography.absolutelyMassive`

Reply:
`Typography.HeftyBoi`